packages/graphback/src/migrations/changes.ts

```diff
@@ -0,0 +1,4 @@
+export enum CHANGES {
```

Comment:
[Naming] Too generic name.
[Comments] Lack of comment "contains changes that will be taken into consideration..

Really good start but we will need to probably go thru the all changes and categorize them into:

- Breaking changes we do not support
- Changes that we support but are ambiguous (TYPE_ADDED - but in fact, it was renamed) 

Document and create tasks on how to implement the 20 from 80 % typical cases that we need to support